region/maridia/inner-pink.json

```diff
@@ -582,6 +612,18 @@
           "locks": null
         }
       ],
+      "enemies": [
```

Comment:
There are requirements to reaching some of those enemies. Only 2 Mochtroids are freely reachable from node 1. One more requires (Morph or grapple) + (suitless or gravity), and the other enemies require gravity or (suitless + (a jump assist or ice)).

Would it make sense to add nodes here?

Reply:
Overall the reqs in this room are weird. Not so much 1 to 2 but 2 to 1. Suitless reqs mixed with a must have canSwim. I can't find any clips you're able to do without canSwim in that direction. For all enemies except 1 Mochtroid a betweenNodes should work since clips or gravity is all you need to collect them. I say we just leave out the one trapped Mochtroid instead of making a node for that single one and fix the 2 to 1 reqs into canSwim variants.

Reply:
I remember messing around a lot in there and maybe I figured out a really tight setup to get up there with nothing? Anyhow I can't do it again.
But probably the more reasonable play is the alter 2 to 1 and add an option of Wave or 2 PBs to the requirements, to break out a Puyo to climb on.

Reply:
Ah right that's a thing